README.md

```diff
@@ -49,6 +49,22 @@ The module exposes a number of endpoints to provide a composite object that link
 * queryParameters supported: No
 * Permissions required?: No
 
+#### `/bl-users/by-id/<id>/open-transactions`
+* Description: Return an object listing number of open transactions that are associated to the user referenced by the user's id
+* Supported operations: GET
+* queryParameters supported: no
```

Comment:
Why is this line included for this endpoint and not the very similar one below?

Reply:
Thanks for catching this! There was no intention behind this. I added the missing line to the similar ones below.